kinde_api/lib/kinde_api/api/environments_api.rb

```diff
@@ -139,6 +139,63 @@ def delete_environement_feature_flag_overrides_with_http_info(opts = {})
       return data, status_code, headers
     end
 
+    # List environment feature flags
+    # Get environment feature flags.
+    # @param [Hash] opts the optional parameters
+    # @return [GetEnvironmentFeatureFlagsResponse]
+    def get_environement_feature_flags(opts = {})
+      data, _status_code, _headers = get_environement_feature_flags_with_http_info(opts)
+      data
+    end
+
+    # List environment feature flags
+    # Get environment feature flags.
+    # @param [Hash] opts the optional parameters
+    # @return [Array<(GetEnvironmentFeatureFlagsResponse, Integer, Hash)>] GetEnvironmentFeatureFlagsResponse data, response status code and response headers
+    def get_environement_feature_flags_with_http_info(opts = {})
+      if @api_client.config.debugging
+        @api_client.config.logger.debug 'Calling API: EnvironmentsApi.get_environement_feature_flags ...'
+      end
+      # resource path
+      local_var_path = '/api/v1/environment/feature_flags'
```

Comment:
if the resource path will stay the same then wouldn't it be better to have a constant initialised for it

Reply:
well, again - it is generated code. and, unlike files like `api_client.rb` and `connection.rb`, these kind of files can be changed in any time with specification change, so we can't replace and modify it easily.

code is not perfect, i understand, but it comes out of openapi generator